src/Integration.Vsix/SonarLintTagger/IssuesSnapshot.cs

```diff
@@ -83,31 +92,70 @@ public override bool TryGetValue(int index, string columnName, out object conten
+                    content = "SonarLint";
                     return true;
 
                 case StandardTableKeyNames.ErrorCode:
                     content = this.issueMarkers[index].Issue.RuleKey;
                     return true;
 
+                case StandardTableKeyNames.ErrorRank:
+                    content = ErrorRank.Other;
+                    return true;
+
+                case StandardTableKeyNames.ErrorCategory:
+                    content = ToString(this.issueMarkers[index].Issue.Type);
+                    return true;
+
                 case StandardTableKeyNames.ErrorCodeToolTip:
+                    content = $"Open description of rule {this.issueMarkers[index].Issue.RuleKey}";
+                    return true;
+
                 case StandardTableKeyNames.HelpLink:
-                    // TODO use correct version
-                    // TODO add JavaScript rules on website
-                    //content = string.Format(CultureInfo.InvariantCulture, "http://www.sonarlint.org/visualstudio/rules/index.html#version=5.9.0.992&ruleId={0}", this.issueMarkers[index].Issue.RuleKey);
-                    content = null;
+                    string ruleKey = this.issueMarkers[index].Issue.RuleKey;
+                    string ruleId = ruleKey.Substring(ruleKey.IndexOf(':') + 1);
+                    string repo = ruleKey.Substring(0, ruleKey.IndexOf(':'));
+                    string language = GetLanguage(repo);
+                    string url = $"http://vs.sonarlint.org/rules/index.html#sonarLintVersion={SonarLintVersion}&ruleId={ruleId}";
+                    if (language != null)
+                    {
+                        url += $"&language={Uri.EscapeDataString(language)}";
+                    }
+                    content = url;
                     return true;
 
                 case StandardTableKeyNames.ProjectName:
-                    // TODO get project name
-
+                    content = projectName;
+                    return true;
                 default:
                     content = null;
                     return false;
             }
         }
 
+        private string GetLanguage(string repo)
+        {
+            switch (repo)
+            {
+                case "javascript": return "JavaScript";
+                case "c": return "C";
+                case "cpp": return "C++";
+                default: return null;
+            }
+        }
+
+        private object ToString(Issue.Types.Type type)
+        {
+            switch (type)
+            {
+                case Issue.Types.Type.Vulnerability: return "Vulnerability";
+                case Issue.Types.Type.Bug: return "Bug";
+                case Issue.Types.Type.CodeSmell:
+                default:
+                    return "Code Smell";
+            }
+        }
+
         public override bool CanCreateDetailsContent(int index)
         {
             // TODO flip to true when detailed description is ready
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)